### ISSUE TITLE
tpa command sometimes does not work

### ISSUE THREAD

**Issue by Dr-WeiAL** [John-Paul-R/Essential-Commands#121]
Version 1.18.2
When accepting the other party's tpa request, the following message will be prompted
No pending teleport request from the specified player.
Supplement: The problem only occurs when you click [Accept]

**Comment by John-Paul-R** [comment]
Hmm... how consistently does this happen? I currently cannot reproduce the issue.

My steps:

1. (As Player1) `/tpa Player2`
2. (As Player2) wait 5-15s (no more)
3. (As Player2) click `[Accept]`
4. Teleport executes.

**Comment by Dr-WeiAL** [comment]
My ID is WeiAL, and I let another player whose ID is SuiXing233 use /tpa WeiAL. After I click [Accept], it will prompt No pending teleport request from the specified player.
Maybe you could consider testing with the same ID as us?

**Comment by Dr-WeiAL** [comment]
Added: At that time the console prompted such a message
  [Essential Commands] Loading PlayerData for player with uuid '38b5c44b-3d7a-4120-ab8e-2506b6094b71'.

**Comment by John-Paul-R** [comment]
How consistently does this happen? Every time, only sometimes?

**Comment by Dr-WeiAL** [comment]
It should be every time, because I see someone /tpa the same person many times in a row and it doesn't work

**Comment by John-Paul-R** [comment]
This is quite strange. To confirm:
- you are running version [0.24.5-mc1.18.2](https://github.com/John-Paul-R/Essential-Commands/releases/tag/0.24.5-mc1.18.2)
- you're clicking `[Accept]` before the request expires (by default, requests expire after 60 seconds)
- running `/tpaccept` _does_ accept the request

Additional question: what happens if you do `/tpaccept <requester_player_name>`? Same error, or does it work?

**Comment by Dr-WeiAL** [comment]
- The version is 0.24.5-mc1.18.2
- The request has not expired
- This is not very clear because no player uses this
Additional question: Clicking `[Accept]` doesn't work, but using `/tpaccept <requester_player_name>` works

Supplement: I only installed this mod on the server side, not on the client side

**Comment by John-Paul-R** [comment]
Weird question: is your server in offline mode?

**Comment by Dr-WeiAL** [comment]
No, this server of mine has online mode and whitelisting enabled
